components/bsx/Gallery/Item/AvailableActions.vue

```diff
@@ -180,6 +180,14 @@ export default class AvailableActions extends mixins(
       balanceInputComponent instanceof BalanceInput
     ) {
       this.isBalanceInputValid = balanceInputComponent.checkValidity()
+      // ad-hoc fix for empty input value
+      if (this.meta === '0') {
+        this.isBalanceInputValid = false
+      }
+    }
+    // ad-hoc fix for empty input value
+    if (balanceInputComponent instanceof BalanceInput && this.meta === '0') {
+      this.isBalanceInputValid = false
     }
```

Comment:
pls delete these duplicate code : )

Reply:
using `git amend` to cause that, sorry for that. removed.